FirebasePerformance/Sources/FPRClient.m

```diff
@@ -340,4 +357,18 @@ - (void)disableInstrumentation {
   [self.configuration setInstrumentationEnabled:NO];
 }
 
+#pragma mark - FIRSessionsSubscriber
+
+- (void)onSessionChanged:(FIRSessionDetails *_Nonnull)session {
+  FPRLogDebug(kFPRSessionId, @"Session ID changed: %@", session.sessionId);
+}
+
+- (BOOL)isDataCollectionEnabled {
+  return self.configuration.isDataCollectionEnabled;
+}
+
+- (FIRSessionsSubscriberName)sessionsSubscriberName {
+  return FIRSessionsSubscriberNameCrashlytics;
```

Comment:
Must be Performance

Reply:
I wanted a way to ensure this and `addDependencyWithName:` used the same name, but couldn't think of any outside defining a constant and use it both places

Reply:
Good catch, I saw "Registering Sessions SDK subscriber with name: Crashlytics, data collection enabled: true" in the logs but then I thought it's normal, Crashlytics is already integrated, turns out it's just my typo